source/renderer/app/api/api.js

```diff
@@ -978,6 +1019,7 @@ export default class AdaApi {
   setSubscriptionStatus: Function;
   setLocalBlockHeight: Function;
   setNetworkBlockHeight: Function;
+  setFaultyNodeSettingsApi: boolean;
```

Comment:
@daniloprates we should not have this test call defined in the main Api.
Can you please move it to the patchAdaApi? Also I think it is better to use const in the patchAdaApi like we did for this case: https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/api/utils/patchAdaApi.js#L119-L121

Reply:
@nikolaglumac  `setFaultyNodeSettingsApi` is defined and used [in the patchAdaApi](https://github.com/input-output-hk/daedalus/pull/1339/files/749277390a1c0b65d592cd1cfd54b690d3a1ee89#diff-298cb1b5c528df77cb1c416241432d87R146), but I couldn't find another way of declaring its type. I've done like other methods from the patchApi, like `setSubscriptionStatus`.

Reply:
Ooooh that's right - I have missed it sorry! You are right :)